src/script/lua_api/l_area_store.cpp

```diff
@@ -158,19 +158,7 @@ int LuaAreaStore::l_insert_area(lua_State *L)
 	LuaAreaStore *o = checkobject(L, 1);
 	AreaStore *ast = o->as;
 
-	Area a;
-
-	a.minedge = check_v3s16(L, 2);
-	a.maxedge = check_v3s16(L, 3);
-
-	a.extremifyEdges();
```

Comment:
Line by line:
Done later
Empty
Done by constructor
Ditto
Empty
Done by constructor

And below:

ID set by insertArea
Empty
Next five lines: No longer necessary since the AreaStore handles IDs.

This does not break the API in any way.  What feature are you saying is removed?